namelists/fesom2/awicm3/awicm-3-master/namelist.io

```diff
@@ -0,0 +1,45 @@
+&diag_list
+ldiag_solver     =.false.
+lcurt_stress_surf=.false.
+ldiag_curl_vel3  =.false.
+ldiag_energy     =.false.
+ldiag_salt3D     =.false.
+ldiag_dMOC       =.false.
+ldiag_DVD        =.false.
+ldiag_forc       =.false.
+/
+
+&nml_listsize
+io_listsize=100 !number of streams to allocate. shallbe large or equal to the number of streams in &nml_list
+/
+
+# for sea ice related variables use_ice should be true, otherewise there will be no output
```

Comment:
This is a FORTRAN namelist, right? I think you need `!` instead of `#`. Be careful

Reply:
In this case it's the namelist I got from the fesom2 source code. Fixed here: https://github.com/FESOM/fesom2/pull/92/files
I'll fix this in the esm-tools after it's merged there.

The idea of this branch is that we can use their namelists directly.